lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -599,6 +599,7 @@ private void informAboutConatinerFailureDuringRestart(final String containerId)
   }
 
   private synchronized void writeToEvaluatorLog(final String entry) throws IOException {
+    /*
```

Comment:
This should be part of another fix for [REEF-85](https://issues.apache.org/jira/browse/REEF-85).